### ISSUE TITLE
Descriptions for groups and snapshots

### ISSUE THREAD

**Issue by captainbrosset** [web-platform-dx/web-features#1141]
We're thinking about publishing groups and snapshots, but (unless I'm mistaken) our groups and snapshots don't have descriptions yet.
For a consumer to use those groups/snapshots in a product, it would make sense for them to have a short description, just like features do.

At least I know I would love to display groups on https://captainbrosset.github.io/web-features-explorer/ as a way to navigate some of the features.

**Comment by ddbeck** [comment]
This is likely to blocked by https://github.com/web-platform-dx/web-features/issues/648 (for groups, at least).

**Comment by captainbrosset** [comment]
Discussed about this on the WebDX CG right now: we haven't heard about a need for descriptions from consumers so far. Closing this.
However, we do have _some_ signals that groups are useful: 
- as an authoring experience (for webdx maintainers)
- on webstatus.dev: https://webstatus.dev/?q=group%3Aperformance for discoverability

So, we should create a new issue to figure out how we want to even do grouping: https://github.com/web-platform-dx/web-features/issues/4018.
